### ISSUE TITLE
hierarchicalSelectors not working

### ISSUE THREAD

**Issue by corysimmons** [stylelint/stylelint#708]
```js
'indentation': [2, {
  hierarchicalSelectors: true
}]
```

![image](https://cloud.githubusercontent.com/assets/224636/12878753/d5035d18-cdf7-11e5-885a-3caafa3a8694.png)

Am I using this the correct way as specified [here](https://github.com/stylelint/stylelint/blob/master/src/rules/indentation/README.md#hierarchicalselectors-truefalse)?

Also, I want to be able to use this feature, but I desperately don't want to enforce spaces or tabs in my config (I feel like that's the job of .editorconfig - not a CSS config). Is there anyway to use hierarchicalSelectors without enforcing an opinion on spaces/tabs?

**Comment by davidtheclark** [comment]
This pattern does in fact work as expected: I can write a passing test with that exact CSS. Which suggests to me that this problem you're seeing lies in linter-stylelint and should be filed there.

If you can reproduce this issue *outside* of Atom, let me know and we'll re-open.

**Comment by corysimmons** [comment]
Does this pattern work without specifying an indentation size? Can stylelint figure it out on it's own or try to grab it from .editorconfig?

**Comment by TrySound** [comment]
@corysimmons Only by options adapter. All configs designed to be encapsulated to prevent any collisions.

**Comment by corysimmons** [comment]
> Only by options adapter

I'm sorry, what is the options adapter here? You mean specifying a indent size (or tab)?

**Comment by TrySound** [comment]
@corysimmons Some code which will parse `.editorconfig` and add and add rule in stylelint.

**Comment by corysimmons** [comment]
Ah, do you happen to know if there is a plugin out there that does this yet?

**Comment by TrySound** [comment]
Nope. You can try to make it by yourself.

**Comment by corysimmons** [comment]
:+1:
